consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/RoundChangeMetadata.java

```diff
@@ -15,30 +15,76 @@
 package org.hyperledger.besu.consensus.qbft.payload;
 
 import org.hyperledger.besu.consensus.common.bft.payload.SignedData;
+import org.hyperledger.besu.consensus.qbft.messagewrappers.RoundChange;
+import org.hyperledger.besu.ethereum.core.Block;
 
 import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
 import java.util.StringJoiner;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Lists;
 
 public class RoundChangeMetadata {
 
-  private final Collection<SignedData<RoundChangePayload>> roundChangePayloads;
-  private final Collection<SignedData<PreparePayload>> prepares;
+  private final Optional<Block> block;
+  private final List<SignedData<RoundChangePayload>> roundChangePayloads;
+  private final List<SignedData<PreparePayload>> prepares;
 
   public RoundChangeMetadata(
```

Comment:
Don't think this needs to change

Reply:
removed, its actually captured in the statemachine PR